routes/api/auth.js

```diff
@@ -1,14 +1,56 @@
 var express = require('express');
+var passport = require('passport')
+var LocalStrategy = require('passport-local').Strategy;
 var router = express.Router();
+const mangoose = require('mongoose');
+const User = mangoose.model('User');
+const { login, logout, isLoggedIn } = require('../../services/auth');
 
 const userService = require('../../services/user');
 
-router.post('/login', function (req, res, next) {
-    res.send('User loggged in successfully');
+var passport = require('passport')
```

Comment:
Move these passport specific initializations to a different file - lib/passport.js or services/passport.js and then include it in app.js itself. This should not be present in the route file.

Reply:
made the changes